package.json

```diff
@@ -22,7 +22,7 @@
     "lint": "yarn lint:sol && yarn lint:js",
     "lint:js": "./node_modules/.bin/eslint .",
     "lint:sol": "./node_modules/.bin/solhint ./contracts/\\*\\*/\\*.sol -w 0",
-    "deploy:contracts": "./node_modules/.bin/hardhat run scripts/deploy/001_initialize_contracts.ts",
+    "deploy:contracts": "env DEBUG='@openzeppelin*,card-protocol.*' ./node_modules/.bin/hardhat run scripts/deploy/001_initialize_contracts.ts",
```

Comment:
Did you intend to commit this change?

Reply:
Hey, yeah this is intentional because I switched from using `console.log` to debug to make logging easier to follow (it color codes it so you can follow what's logged from our code vs what's logged from libraries)